.circleci/config.yml

```diff
@@ -654,14 +657,33 @@ workflows:
 
   nightly:
     jobs:
-      - nightly-owasp-scan
+      - nightly-owasp-scan:
+          target_env: develop
+          filters:
+            branches:
+              only:
+                - develop
+      - nightly-owasp-scan:
+          target_env: staging
+          filters:
+            branches:
+              only:
+                - main
+      - nightly-owasp-scan:
+          target_env: prod
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
Depending on the branch a different parameter will be sent as target_env.

I have a plan to refactor this sort of logic to make it shorter. Will do that for multiple parts of the ci in the future.